src/main/java/com/microsoftopentechnologies/windowsazurestorage/AzureStorageBuilder.java

```diff
@@ -366,5 +422,19 @@ public StorageAccountInfo getStorageAccount(String storageAccountName) {
 			}
 			return storageAcc;
 		}
+
+		public DescriptorExtensionList<BuildSelector, Descriptor<BuildSelector>> getAvailableBuildSelectorList() {
```

Comment:
Something seems to be going wrong here.  For only a single AzureStorageBuilder, the removed BuildSelectors are not present, but if I add two AzureStorageBuilder steps, all of the BuildSelectors are available and are persisted to the job config if saved.  On reload, the options are still present in the list but the selected option has been reset.